FreeRTOS-Plus/Source/FreeRTOS-Plus-CLI/FreeRTOS_CLI.c

```diff
@@ -158,8 +160,8 @@ size_t xCommandStringLength;
 		/* Search for the command string in the list of registered commands. */
 		for( pxCommand = &xRegisteredCommands; pxCommand != NULL; pxCommand = pxCommand->pxNext )
 		{
-			pcRegisteredCommandString = pxCommand->pxCommandLineDefinition->pcCommand;
-			xCommandStringLength = strlen( pcRegisteredCommandString );
+			const char *pcRegisteredCommandString = pxCommand->pxCommandLineDefinition->pcCommand;

```

Comment:
Declaring the local variable in the beginning is a good practice. It also provides better portability for all compilers.